packages/demo-gatsby/src/pages/about.js

```diff
@@ -0,0 +1,75 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+import React from "react"
+import { graphql } from "gatsby"
+import { useLocalMdx } from "gatsby-tinacms-mdx"
```

Comment:
based on https://github.com/tinacms/tinacms/pull/713#discussion_r375002577 shouldn'tit be `useLocalMdxForm` now?

Reply:
yep! It's been fixed in the original PR I believe